fortigate_exporter.go

```diff
@@ -118,6 +131,10 @@ func probeHandler(w http.ResponseWriter, r *http.Request) {
 func main() {
 	flag.Parse()
 
+	version, hash := getVersion()
+	log.Printf("FortigateExporter:\tversion: %s\t\tcommit: %s", version, hash)
```

Comment:
A bit unrelated, but I would like us to switch to using uber's Zap logging in the future to allow for optional structured logs - the normal human logs are much nicer as well. This is fine for now though.